conduit/data/datasets/audio/ecoacoustics.py

```diff
@@ -98,8 +92,10 @@ def __init__(
         self.root = Path(root).expanduser()
         self.download = download
         self.base_dir = self.root / self._BASE_FOLDER
-        self.labels_dir = self.base_dir / self.INDICES_DIR
-        self._processed_audio_dir = self.base_dir / self._PROCESSED_DIR
+        self.labels_dir = self.base_dir / self._INDICES_FILE_INFO.name[:-4] / self.INDICES_DIR
+        # target directory needs to depend on the preprocessing function
+        preprocess_id = re.sub(r"\W+", '', str(preprocessing_transform))
```

Comment:
```suggestion
        preprocess_id = preprocessing_transform.__class.__name__
```
Or something like this; perhaps you'd like to convert to snake case

Reply:
I was thinking that `str(preprocessing_transform)` will show more information than just the class name -- like FFT windows size -- (and so allows for more disambiguation), but it doesn't actually seem so, so the approach you're suggesting here should be fine.